test/functional/test_ipmi_kcs.py

```diff
@@ -14,6 +14,8 @@
     - local sel can work
     - local sdr can work
     - local user can work
+Test SMBIOS data
```

Comment:
file name is testing KCS which this inconsistent with the test content. Suggest to either change file name or move the SMBIOS related test to another file.